src/main/java/gov/loc/repository/bagit/writer/PayloadWriter.java

```diff
@@ -21,73 +21,80 @@
  * Responsible for writing out the bag payload to the filesystem
  */
 public final class PayloadWriter {
+
```

Comment:
As far as I can tell, this file only contains formatting changes, is that correct?